website/docs/reference/commands/clone.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "About dbt clone command"
+sidebar_label: "clone"
+id: "clone"
+---
+
+The `dbt clone` command clones selected nodes from the [specified state](/reference/node-selection/syntax#establishing-state) to the target schema(s). You must always use different directories for source and target otherwise you will get an error message. This command makes use of the `clone` materialization:
```

Comment:
> You must always use different directories for source and target otherwise you will get an error message.

Agh, on second read, the presence of this sentence so early in the page does confuse me. Let's cut it for now, and make sure to open the `dbt-core` follow-up issue for better error handling.